packages/core/src/shared/constants.ts

```diff
@@ -189,6 +189,14 @@ export const amazonQHelpUrl = 'https://aws.amazon.com/q/'
 export const amazonQVscodeMarketplace =
     'https://marketplace.visualstudio.com/items?itemName=AmazonWebServices.amazon-q-vscode'
 
+// URLs for Amazon EventBridge Scheduler
+export const eventBridgeSchedulerCreateScheduleUrl: string =
+    'https://docs.aws.amazon.com/scheduler/latest/UserGuide/schedule-types.html'
+export const eventBridgeSchedulerApiGatewayUrl: string = 
+    'https://docs.aws.amazon.com/scheduler/latest/UserGuide/managing-schedule.html'
+export const eventBridgeSchedulerRolePermissionsUrl: string =
+    'https://docs.aws.amazon.com/scheduler/latest/UserGuide/setting-up.html'
+
 /**
  * Names of directories relevant to the crash reporting functionality.
  *
```

Comment:
The addition of EventBridge Scheduler URLs to the constants file is a good practice. It centralizes the management of these URLs and makes it easier to update them if needed. However, there are a couple of suggestions for improvement:

1. Consider adding a prefix to these constants to group them together and make their purpose clearer.

2. It might be helpful to add a comment explaining the purpose of each URL for better maintainability.
```suggestion
// URLs for Amazon EventBridge Scheduler
export const EBS_CREATE_SCHEDULE_URL: string =
    ''
export const EBS_MANAGE_SCHEDULES_URL: string = 
    ''
export const EBS_ROLE_PERMISSIONS_URL: string =
    ''

/**
 * Names of directories relevant to the crash reporting functionality.
 */
```